app/javascript/components/common/AvatarGroup.tsx

```diff
@@ -1,30 +1,30 @@
 import React from 'react'
 import { Avatar } from '../common'
+import { CreditsUsersProp } from './Credits'
 
 export type AvatarGroupProps = {
-  avatarUrls: readonly string[]
-  max?: number
+  users: CreditsUsersProp[]
+  overflow: number
   className?: string
 }
 
 export function AvatarGroup({
-  avatarUrls,
-  max = 0,
+  users,
+  overflow,
   className,
 }: AvatarGroupProps): JSX.Element {
   return (
-    <div className={`c-faces --static ${className}`}>
-      {avatarUrls
-        .slice(0, max > 0 ? max : avatarUrls.length)
-        .map((avatarUrl) => (
-          <Avatar
-            className="face"
-            src={avatarUrl}
-            key={`${avatarUrl}${Math.random()}`}
-          />
-        ))}
-      {max > 0 && avatarUrls.length - max > 0 ? (
-        <div className="counter">+{Math.min(99, avatarUrls.length - max)}</div>
+    <div className={`c-faces-with-overflow-counter ${className}`}>
+      {users.slice(0, 2).map((user) => (
```

Comment:
You're guaranteed that users only ever has a maximum of three elements, so the slice is not needed.